demo-vaadin-8/pom.xml

```
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+    <artifactId>appsec-kit</artifactId>
+    <groupId>com.vaadin</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+
+	<groupId>com.vaadin.appsec-kit</groupId>
+	<artifactId>demo-vaadin-8</artifactId>
```

I didn't notice this before, but let's use the common `com.vaadin` group-id for all the modules, as on other projects. And since we're doing this, let's rename also the module to `appsec-kit-demo-v8` to use the same convention of all modules starting with `appsec-kit`:

```suggestion
	<groupId>com.vaadin</groupId>
	<artifactId>appsec-kit-demo-v8</artifactId>
```

See other suggestions for according changes.